scripts/apps/search/components/MarkedDesksList.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {
+    closeActionsMenu
+} from 'apps/search/helpers';
+
+export class MarkedDesksList extends React.Component {
```

Comment:
We should try and commit to documenting everything. This component has been omitted.